src/views/Home.vue

```diff
@@ -149,6 +169,40 @@ export default {
     retry3Times(sideFederation.methods.getMembers().call).then(
       members => (data.sideFedMembers = members),
     )
+
+    // NFT
+    if (rskConfig.nftBridge == undefined || sideConfig.nftBridge == undefined) {
+      return
+    }
+
+    const rskBridgeNft = new rskWeb3.eth.Contract(NFT_BRIDGE, rskConfig.bridge)
+    rskBridgeNft.methods
+      .getFixedFee()
+      .call()
+      .then(fee => {
+        debugger
+        data.rskFeeNft = fee / rskConfig.feePercentageDivider
+      })
+      .catch(err => {
+        // eslint-disable-next-line no-console
+        console.log('Error in getFixedFee', err)
+      })
+
+    const sideBridgeNft = new sideWeb3.eth.Contract(NFT_BRIDGE, sideConfig.bridge)
+    retry3Times(sideBridgeNft.methods.getFixedFee().call).then(fee => {
+      data.sideFeeNft = fee / sideConfig.feePercentageDivider
+    })
+
+    const NFT_FIXED_CONFIRMATIONS_BLOCK = 3 // maybe we are going to define it into the nft bridge contract
```

Comment:
I'd maybe add a Github issue / Jira ticket under 'technical debt' to not lose sight of this comment